src/components/shared/Slider/Slider.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { Slider } from './Slider';
+
+describe('Stars score component', () => {
+  afterEach(cleanup);
+
+  it('renders stars with score 0', () => {
+    // Act
+    const mockedFunction = () => null;
```

Comment:
In this test or another, it would be nice to test how this function is called.

Reply:
Yes, it could be nice, I have to find a way to shoot an onChange event in this element. 👍